master/internal/db/postgres.go

```diff
@@ -250,6 +252,19 @@ func (db *PgDB) RawQuery(queryName string, params ...interface{}) ([]byte, error
 	return db.rawQuery(db.queries.getOrLoad(queryName), params...)
 }
 
+func (db *PgDB) GetWorkspace(ctx context.Context, id int32) (*workspacev1.Workspace, error) {
+	w := &workspacev1.Workspace{}
```

Comment:
@stoksc @ioga What do you think about using the proto object directly as the bun model?

Pros I see:
- It's simpler, at least when it works.

Cons I see:
- It violates the separation of user api from data model implementation

Reply:
i think we shouldn't at all. it will not work in general and is more susceptible to weird bugs (weird fields aren't mapped right/are mapped at all). separate models are inevitable the first time the models diverge, so just make them separate, otherwise the first time they diverge may just result in a bug.

Reply:
also, you can't change generated models so you can't embed a `bun.BaseModel` in it. does this even work?

Reply:
I agree with @stoksc , that's not going to work as a general solution.

Reply:
https://github.com/determined-ai/ci-shortcuts/blob/master/cibuild/models.go#L28-L53 is the pattern we should be using, imo